src/payload/collections/PodcastEpisodes/hooks/populateContributors.ts

```diff
@@ -0,0 +1,25 @@
+import type { AfterReadHook } from "payload/dist/collections/config/types";
+
+export const populateContributors: AfterReadHook = async ({ doc, req: { payload } }) => {
```

Comment:
Why do we need this? It would be better to deal with this in a seperate PR, where you can explain better why it's needed/what it does

Reply:
I don't think contributors needs to be private. They are basically authors and we wont be keeping any sensitive info on them

Reply:
i agree, i'll change read access to "anyone"

Reply:
done ✅